src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -290,14 +290,27 @@ internal static void SetupCallbacks(TraceEventDispatcher source)
                 var clrRundown = new ClrRundownTraceEventParser(source);
                 clrRundown.RuntimeStart += doAtRuntimeStart;
                 source.Clr.RuntimeStart += doAtRuntimeStart;
+                Dictionary<TraceProcess, TraceLoadedDotNetRuntime> processRuntimes = null;
 
                 CircularBuffer<ThreadWorkSpan> RecentThreadSwitches = new CircularBuffer<ThreadWorkSpan>(1000);
                 source.Kernel.ThreadCSwitch += delegate (CSwitchTraceData data)
                 {
```

Comment:
@jeffschwMSFT I updated the cache to apply to most uses of this data inside `SetupCallbacks`. The exception is the use of `MutableTraceEventStackSource` and `HasMutableTraceEventStackSource` which operate on a slightly different object and don't currently appear to be a problem.